front-end/src/components/UI/Hamburger/NavBar/NavButton/NavButton.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cl from './NavButton.module.css'
+import {useNavigate} from "react-router-dom";
+
+const NavButton = ({text, path="/", callback}) => {
```

Comment:
Could we make something like `RouterButton`, which we can use not only in navbar?

Reply:
This makes no sense. Navbar should be the only way to navigate